src/nvim/os/unix_defs.h

```diff
@@ -31,12 +31,6 @@
 #ifndef USR_EXRC_FILE
 # define USR_EXRC_FILE "~/.exrc"
 #endif
-#ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE "~/.nvimrc"
```

Comment:
@justinmk Windows is out of scope of XDG Basedir, follow Windows' standards for Windows instead.  For example, Vala's `Environment.get_user_config_dir()` [uses `CSIDL_LOCAL_APPDATA` for Windows](http://valadoc.org/#!api=glib-2.0/GLib.Environment.get_user_config_dir).

What you want to do about Windows however is up to you, 
thank you for working on this btw. ❤